### PR TITLE
Feature/oddfyield

### PR DESCRIPTION
**Pull request by haalEpplus**
Added two new functions, oddfprice & oddfyield, including unit-tests for both functions.

### CODE REVIEW

File: src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OfficeOpenXml.FormulaParsing.Excel.Functions.Finance.Implementations
+{
+    internal class OddfpriceImpl
+    {
+        private readonly System.DateTime _settlementDate;
```
Comment (JanKallman) [inline src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs:24]: Add a `using System;` in the top of the file  and remove the namespace references on `DateTime`

File: src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+            {
+                return new FinanceCalcResult<double>(eErrorType.Num);
+            }
+            var sDate = FinancialDayFactory.Create(_settlementDate, _basis);
+            var mDate = FinancialDayFactory.Create(_maturityDate, _basis);
+            var fcDate = FinancialDayFactory.Create(_firstCouponDate, _basis);
+            var iDate = FinancialDayFactory.Create(_issueDate, _basis);
+            var daysDefinition = FinancialDaysFactory.Create(_basis);
+
+            var A = daysDefinition.GetDaysBetweenDates(iDate, sDate);
```
Comment (JanKallman) [inline src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs:70]: Some of the variable names below are not that describing, like A, E, N, DFC, T1,T2,T3,T4 etc. Is it possible to give them more reader friendly names? Optionally add a comment describing what they are.

File: src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs (hunk 1 above)
Comment (haalEpplus) [reply src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs:24]: In Epplus formulaparser there already is a namespace called DateTime (fficeOpenXml.FormulaParsing.Excel.Functions.DateTime), it collides with System.DateTime. To resolve we would need to change that namespace, but that is maybe not in the scope of this pull request?

File: src/EPPlus/FormulaParsing/Excel/Functions/Finance/Datetime.cs
[the remark this answers is not in the document]
Comment (haalEpplus) [reply src/EPPlus/FormulaParsing/Excel/Functions/Finance/Datetime.cs:1]: Fixed in the most recent commit

File: src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs (hunk 2 above)
Comment (haalEpplus) [reply src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs:73]: Added comment that refers to the formula in the Microsoft documentation

File: src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs (hunk 1 above)
Comment (JanKallman) [reply src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs:24]: Mats will rename this namespace in another commit.

File: src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs (hunk 2 above)
Comment (JanKallman) [reply src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs:73]: Great!

**Pull request merged**